### DISCUSSION TITLE
Welcome to xeokit-sdk Discussions!

### DISCUSSION THREAD

**Discussion by xeolabs** [xeokit/xeokit-sdk#1407] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inv-praveen** [comment]
Can I get assistance to integrate it in angular

**Reply by inv-praveen** [reply]
@xeolabs Lindsay Kay

**Reply by xeolabs** [reply]
Hi @inv-praveen , if your enterprise needs assistance implementing your application on xeokit, we'd be happy to help - if so, please get in touch here: https://creoox.com/en/contact/

**Comment by TobxtR** [comment]
Hi, I am Toby and I am experimenting with xeokit-bimviwer

I am wondering if there are ways to instantiate/spawn in Models or simple shapes during runtime
Please assist me in this particular question @xeolabs
